WORKSPACE

```diff
@@ -28,3 +29,18 @@ yarn_install(
 load("@npm//:install_bazel_dependencies.bzl", "install_bazel_dependencies")
 
 install_bazel_dependencies()
+
+jvm_maven_import_external(
+    name = "error_prone_annotations",
+    artifact = "com.google.errorprone:error_prone_annotations:2.3.0",
+    artifact_sha256 = "524b43ea15ca97c68f10d5f417c4068dc88144b620d2203f0910441a769fd42f",
+    licenses = ["notice"],  # Apache 2.0
+    server_urls = ["https://repo1.maven.org/maven2"],
+)
+
+http_archive(
+    name = "intellij_ce_2019_3",
+    build_file = "@//third_party/intellij_sdk:BUILD.idea193",
+    sha256 = "fb347c3c681328d11e87846950e8c5af6ac2c8d6a7e56946d3a10e6121d322f9",
+    url = "https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/idea/ideaIC/2019.3.2/ideaIC-2019.3.2.zip",
```

Comment:
Making sure people have the right version of intellij could be fun

Reply:
Yah I don't really know the compatibility here, I just went with the version I have installed now. It's probably worth checking. These deps are set to `neverlink` so they don't make it into the final deploy jar, so just depends if the IJ API has changed I guess